tools/cooler/cooler_zoomify.xml

```diff
@@ -0,0 +1,47 @@
+<tool id="cooler_zoomify" name="cooler_zoomify" version="@VERSION@+galaxy0" profile="18.01">
+  <description>Copy and balance a cool file.</description>
```

Comment:
```suggestion
  <description>Copy and balance a cool file</description>
```

Reply:
The text is wrong isn't it? It should be "Generate a multi-resolution cooler file by coarsening"?